packages/components/src/components/hds/side-nav/header/icon-button.ts

```diff
@@ -18,11 +18,27 @@ interface HdsSideNavHeaderIconButtonSignature {
 }
 
 export default class HdsSideNavHeaderIconButtonComponent extends Component<HdsSideNavHeaderIconButtonSignature> {
-  /**
-   * @param ariaLabel
-   * @type {string}
-   * @description The value of `aria-label`
-   */
+  constructor(
+    owner: unknown,
+    args: HdsSideNavHeaderIconButtonSignature['Args']
+  ) {
+    super(owner, args);
+
+    deprecate(
+      'The `Hds::SideNav::Header::IconButton` sub-component is now deprecated and will be removed in the next major version of `@hashicorp/design-system-components`. Use `Hds::Button` with the `isIconOnly` variant instead.',
+      false,
+      {
+        id: 'hds.components.sidenav.header.iconbutton',
+        until: '5.0.0',
+        url: 'https://helios.hashicorp.design/components/side-nav?tab=version%20history#4100',
```

Comment:
it looks like we're missing the actual content that should appear on the version tab outlining the changes here

Reply:
Could you explain a bit more? Do I manually add an .md file in "website/docs/components/side-nav/partials/version-history/" like the "4.8.0.md" in there now?

Do I copy the changeset text into it (pasted below)?

"Deprecated the component. Use the `Hds::Button` component with `isIconOnly` set to `true` as a replacement."

Reply:
That sounds roughly correct to me. You should be able to mimic what Zack did in https://github.com/hashicorp/design-system/pull/2347/files if you're looking for a reference.

Reply:
Does this look correct? https://hds-website-git-hds-3770-deprecate-sidenav-iconbutton-hashicorp.vercel.app/components/side-nav?tab=version%20history